napi-inl.h

```diff
@@ -2697,11 +2716,11 @@ inline FunctionReference& AsyncWorker::Callback() {
 }
 
 inline void AsyncWorker::OnOK() {
-  _callback.MakeCallback(_receiver.Value(), {});
+  _callback.MakeCallback(_receiver.Value(), {{}});
```

Comment:
This should have been changed in the latest revision.  This actually had side-effects that I encountered during testing.